From 6f9a51a7ec69cf8f8b1e3f8a247a1c511141bf42 Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Mon, 22 May 2023 11:12:40 +0800
Subject: [PATCH 5/9] amlbian: load amlogic-cameral.ko in ramfs [1/1]

PD#SWPL-124626

Problem:
kernel panic when loading amlogic-cameral.ko.

Solution:
load amlogic-cameral.ko in ramfs.

Verify:
local

Signed-off-by: yi.liu <yi.liu@amlogic.com>
Change-Id: I5141622fc32a5afe0f48c5e950469bb49ee1c89f
---
 archives/filesystem/special/amlogic/A311D-W400/etc/modules   | 1 -
 archives/filesystem/special/amlogic/S905D3-AC200/etc/modules | 1 -
 2 files changed, 2 deletions(-)

diff --git a/archives/filesystem/special/amlogic/A311D-W400/etc/modules b/archives/filesystem/special/amlogic/A311D-W400/etc/modules
index a96f766..f89cd74 100644
--- a/archives/filesystem/special/amlogic/A311D-W400/etc/modules
+++ b/archives/filesystem/special/amlogic/A311D-W400/etc/modules
@@ -1,4 +1,3 @@
-amlogic-camera
 amlogic-crypto-dma
 amlogic-usb
 amlogic-wireless
diff --git a/archives/filesystem/special/amlogic/S905D3-AC200/etc/modules b/archives/filesystem/special/amlogic/S905D3-AC200/etc/modules
index a96f766..f89cd74 100644
--- a/archives/filesystem/special/amlogic/S905D3-AC200/etc/modules
+++ b/archives/filesystem/special/amlogic/S905D3-AC200/etc/modules
@@ -1,4 +1,3 @@
-amlogic-camera
 amlogic-crypto-dma
 amlogic-usb
 amlogic-wireless
-- 
2.25.1

